addons/sourcemod/scripting/ConsoleChatManager.sp

```diff
@@ -546,9 +546,19 @@ public Action SayConsole(int client, const char[] command, int args)
 	return Plugin_Handled;
 }
 
+stock int MyCharToLower(int c)
```

Comment:
maybe put it in utilshelper also check whats in smlib maybe we can directly merge them or the functions are already available?